remote/webby/webby_unix.h

```diff
@@ -19,7 +19,7 @@ static int wb_socket_error(void)
 
 static int wb_valid_socket(webby_socket_t socket)
 {
-  return socket > 0;
+  return socket >= 0;
```

Comment:
@bbi-markslemko should this be guarded in an aarch64 preprocessor check so that we still consider 0 to be an invalid socket in other platforms?

Reply:
I was wondering why is the socket file descriptor (zero) 0 ever? That is odd. 

socket docs:
>        On success, a file descriptor for the new socket is returned.  On
>        error, -1 is returned, and errno is set appropriately.

however `webby_socket_t` is an `unsigned int`, so this would/should always succeed and make the `wb_valid_socket` test pointless.

I think this is not where the problem is, but it is interesting that this would allow ImGUI to work.

Reply:
given nn::socket::Socket
> Returns
> Returns -1 on error. If it succeeds, it returns a non-negative integer that is a descriptor for the created socket.
it might be more appropriate to bracket in #if defined(__NN_AARCH64__) and just return true

Reply:
webby_socket_t is an int on NX and unsigned int on Windows for example, so different behavior

Reply:
ifdef'd to avoid any side effects!